packages/app-desktop/gui/NoteSearchBar.jsx

```diff
@@ -164,12 +167,17 @@ class NoteSearchBarComponent extends React.Component {
 						onKeyDown={this.searchInput_keyDown}
 						ref="searchInput"
 						type="text"
-						style={{ width: 200, marginRight: 5, backgroundColor: this.backgroundColor, color: theme.color }}
+						style={{
+							width: 200, marginRight: 5,
+							backgroundColor: this.backgroundColor,
+							color: theme.color,
+						}}
```

Comment:
Was this change the result of something you did? or was it from the Auto formatting tool that Joplin uses? If you made the change, please revert it.

Reply:
I changed it when I was trying to change the borderColor like in Firefox warning.

I'll revert it.